src/App.js

```diff
@@ -1,20 +1,23 @@
-/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
-import Navigation from './components/Navigation';
+import Footer from './components/Footer';
+import Navbar from './components/Navigation';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
 
-class App extends React.Component {
-  render() {
-    return (
-      <div>
-        <header><Navigation /></header>
-        <main className="main-container">
-          <Calculator />
-        </main>
-        <footer className="footer-container" />
-      </div>
-    );
-  }
+function App() {
+  return (
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED: ** For using ES6 and [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), I think it would be better if you update this function to be `Arrow function` instead of a  `Normal function`.